### PR TITLE
Hra dashboard application

### PR DESCRIPTION
**Pull request by axdanbol**

### CODE REVIEW

**Comment by bherr2** [comment]
@axdanbol there is a merge conflict

File: apps/dashboard-ui/src/app/nx-welcome.component.ts
Comment (axdanbol) [inline apps/dashboard-ui/src/app/nx-welcome.component.ts:1]: Delete this file

File: apps/dashboard-ui/src/app/components/header/header.component.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<div class="header">
+  <div class="logos">
+    <a href="https://humanatlas.io/" target="_blank">
```
Comment (axdanbol) [inline apps/dashboard-ui/src/app/components/header/header.component.html:3]: Add `rel="noopener noreferrer"`

**Review by axdanbol — COMMENTED** [review]
@bhushankhope Tweaks

**Review by bherr2 — APPROVED** [review]

**Pull request merged**